components/dialog/stories/template.js

```diff
@@ -6,49 +6,68 @@ import { Template as Underlay } from "@spectrum-css/underlay/stories/template.js
 import { useArgs } from "@storybook/preview-api";
 import { html } from "lit";
 import { classMap } from "lit/directives/class-map.js";
+import { styleMap } from "lit/directives/style-map.js";
 import { ifDefined } from "lit/directives/if-defined.js";
 import { when } from "lit/directives/when.js";
 
 import "../index.css";
 
 export const Template = ({
 	rootClass = "spectrum-Dialog",
-	isDismissable = true,
+	isDismissible = true,
+	hasDivider = true,
 	isOpen = true,
 	showModal = false,
 	heading,
 	content = [],
 	customClasses = [],
 	id,
+	size = "medium",
+	layout,
+	hasHeroImage = false,
+	heroImageUrl,
+	customStyles = {},
 	...globals
 }, context) => {
 	const [, updateArgs] = useArgs();
-	const scale = window.__scale;
 
 	const Dialog = html`
 		<div
 			class=${classMap({
 				[rootClass]: true,
-				[`${rootClass}--${scale}`]: true,
-				[`${rootClass}--dismissable`]: isDismissable,
+				[`${rootClass}--${size}`]: typeof size !== "undefined", 
+				[`${rootClass}--dismissable`]: isDismissible,
+				[`${rootClass}--${layout}`]: typeof layout !== "undefined",
+				[`${rootClass}--noDivider`]: !hasDivider,
 				...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
 			})}
 			id=${ifDefined(id)}
 			role="dialog"
 			tabindex="-1"
 			aria-modal="true"
+			style=${ifDefined(styleMap(customStyles))}
 		>
 			<div class="${rootClass}-grid">
-				${when(heading, () => [
-					html`<h1 class="${rootClass}-heading">${heading}</h1>`,
-					Divider({
+				${when(hasHeroImage, () =>
+					html`
+						<div 
+							class="spectrum-Dialog-hero"
+							style="background-image:url(${heroImageUrl ?  heroImageUrl : "example-card-portrait.png"})">
+						</div>
+					`
+				)}
+				<h1 class="${rootClass}-heading">${heading}</h1>
+				${when(hasDivider, () =>
+				Divider({
 						horizontal: true,
 						customClasses: [`${rootClass}-divider`],
 						...globals,
-					}, context),
-				])}
-				<section class="${rootClass}-content">${content.map((c) => (typeof c === "function" ? c({}, context) : c))}</section>
-				${when(isDismissable, () =>
+					}),
```

Comment:
```suggestion
					}, context),
```

Missing the context passthrough in this update.

Reply:
Good catch! That should be fixed here: https://github.com/adobe/spectrum-css/pull/2833/commits/95dfddcfd56af9959011fea2b855702e96ccadbc